appcelerator/ioslib
# [TIMOB-18633] Updating for watch support

[TIMOB-18633](https://jira.appcelerator.org/browse/TIMOB-18633)

#### Testing instructions:
* Look in the README.md under the `Running Tests` section for instructions on running the unit tests
* Run the unit tests for simulator `npm run-script test-simulator` before pulling down this PR
* Notice the tests that fail
* Pull down this PR
* Run the tests again
* Notice that 1 more test has been added and more tests are passing (there are still 3 tests in the test-simulator suite that were failing before the PR and are still failing)
